specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2015-12-01-preview/examples/Operations/List.json

```diff
@@ -114,6 +114,15 @@
               "description": "Delete any VMExtension"
             }
           },
+          {
```

Comment:
I dotn think this is correct. 2015 apiversion operations api doesnt support scaleunit

Only the newer api version should return the scalueunits

Reply:
This is a problem with our Operations API. We currently return all operations for any supported API version. We need to make a code change to update the operations controller to be per API version. I will create a bug to fix this.